schema/ATSpectrograph.yaml

```diff
@@ -29,26 +29,87 @@ properties:
     decription: Tolerance for positioning the linear stage.
     type: number
     default: 0.01
+
   filters:
-    decription: Currently installed filter names.
-    type: array
-    uniqueItems: true
-    items:
-      type: string
-    default:
-     - empty_1
-     - empty_2
-     - diffuser
-     - empty_3
+    type: object
+    properties:
+      filter_name:
+        decription: Currently installed filter names.
+        default:
+          - empty_1
+          - empty_2
+          - diffuser
+          - empty_3
+        type: array
+        uniqueItems: true
+        items:
+          type: string
+        minItems: 4
+        maxItems: 4
+
+      filter_central_wavelength:
+        description: >
+          Wavelength for which optical system will be opimized in units of nm. Approximations are sufficient as the
+          focus dependence on wavelength is weak.
+        type: array
+        items:
+          type: number
+        minItems: 4
+        maxItems: 4
+        default:
+          - 700
+          - 701
+          - 702
+          - 703
+
+      filter_focus_offset:
+        description: >
+          Focus offset to be applied on the secondary mirror in units of um, relative to no glass being installed.
+          Positive values push the secondary down and increase the back focal distance, therefore adding glass
+          thickness will result in positive focus offsets.
+        type: array
+        uniqueItems: false
+        items:
+          type: number
+        minItems: 4
+        maxItems: 4
+        default:
```

Comment:
Should't the default be 0.0 for all? why have default 0, 1, 2, 3?

Reply:
I did this because it's easier to find errors in testing. If you swap a value or were to grab the wrong one and they were all zeros then you wouldn't know.